radixdlt-java/src/main/java/com/radixdlt/client/application/RadixApplicationAPI.java

```diff
@@ -825,6 +768,62 @@ public UnsignedAtom buildAtomWithFee(List<ParticleGroup> particleGroups) {
 		return new UnsignedAtom(new Atom(allParticleGroups, metaData));
 	}
 
+	/**
+	 * Represents an atomic transaction to be committed to the ledger
+	 */
+	public final class Transaction {
```

Comment:
So.. an atom??

Reply:
I fear saying "Transaction" might be confusing to people, since database transaction != financial transaction and it's often confused in DLTs

Reply:
I associate this `Transaction` with a financial transaction and I think it works for me and the use-case I had in mind. Then again; maybe I'm just a special kind of stupid who got lucky with a "mental double negation" this time...